lib/link.js

```diff
@@ -11,8 +11,16 @@ export default class Link extends Component {
 
   static propTypes = {
     children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.element
+      PropTypes.element,
+      (props, propName) => {
```

Comment:
> Don't `console.warn` or throw, as this won't work inside `oneOfType`.

https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes

I wonder what this means 🤔